<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDeliveryOrderDlg</class>
 <widget class="QDialog" name="CreateDeliveryOrderDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBarWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>42</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="skinWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#skinWidget{
background-color: rgb(81, 80, 80);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>22</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>22</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="ReceiveProviderForCreateWidget" name="providerPage"/>
        <widget class="ReceiveGoodsForCreateWidget" name="goodsPage"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header>TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReceiveProviderForCreateWidget</class>
   <extends>QWidget</extends>
   <header>ReceiveProviderForCreateWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReceiveGoodsForCreateWidget</class>
   <extends>QWidget</extends>
   <header>ReceiveGoodsForCreateWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
